kubetest/e2e.go

```diff
@@ -220,11 +226,8 @@ func run(deploy deployer, o options) error {
 
 	// TODO(bentheelder): consider remapping charts, etc to testCmd
 
-	var kubemarkWg sync.WaitGroup
-	var kubemarkDownErr error
 	if o.kubemark {
-		kubemarkWg.Add(1)
-		go kubemarkDown(&kubemarkDownErr, &kubemarkWg)
+		errs = util.AppendError(errs, kubemarkDown(dump))
```

Comment:
This broke the optimization that we tear down kubemark in parallel with tearing down the main cluster. - please revert the WaitGroup.

Reply:
Done.